cmd/tailscale/cli/set.go

```diff
@@ -76,6 +79,11 @@ func newSetFlagSet(goos string, setArgs *setArgsT) *flag.FlagSet {
 	setf.BoolVar(&setArgs.updateApply, "auto-update", false, "automatically update to the latest available version")
 	setf.BoolVar(&setArgs.postureChecking, "posture-checking", false, "HIDDEN: allow management plane to gather device posture information")
 	setf.BoolVar(&setArgs.runWebClient, "webclient", false, "run a web interface for managing this node, served over Tailscale at port 5252")
+	dap := tailfs.DefaultAutomountPath()
+	if !version.IsSandboxedMacOS() && dap != "" {
+		setf.BoolVar(&setArgs.automountEnabled, "automount-enabled", false, "enable automatically mounting Tailscale shares")
```

Comment:
This is a profile-specific setting? Does it then unmount on profile switch?

Reply:
I wasn't intending it to be profile specific. Since the root of the directory tree is the tailnet, if you switch tailnets, it'll be reflected in the directory structure of the existing mount point.

Reply:
I'm starting to think these shouldn't be a pref.

Reply:
Alternative suggestions?